nuxeo/values.yaml

```diff
@@ -258,6 +258,19 @@ customProperties: {}
 #     key1=value1
 #     key2=value2
 
+## Custom contributions to be moved to nxserver/config.
+## This is a map to allow mounting XML contributions within the Nuxeo container.
+## They keys MUST represent a valid Nuxeo contribution file, for instance it should end with -config.xml.
+customContributions: {}
+#  path-segment-config.xml: |
+#    <?xml version="1.0"?>
+#    <component name="override-path-segment-maxsize">
+#      <require>org.nuxeo.ecm.core.api.properties</require>
+#      <extension target="org.nuxeo.runtime.ConfigurationService" point="configuration">
+#        <property name="nuxeo.path.segment.maxsize">96</property>
+#      </extension>
+#    </component>
+
```

Comment:
If you could also update the [Parameters](https://github.com/nuxeo/nuxeo-helm-chart?tab=readme-ov-file#parameters) section of the README, I've tried to keep it up-to-date.